_includes/footer.html

```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+    <div class="container">
+        <p class="footer-content">&copy;2021 Copyright. Mention of FB AI?</p>
```

Comment:
It looks like this question, 'Mention of FB AI?', has been removed from figma
plz remove